### PR TITLE
Ret 1131 - task list - employment details - working and notice

### PR DESCRIPTION
**Pull request by lvo700**
https://tools.hmcts.net/jira/browse/RET-1130
https://tools.hmcts.net/jira/browse/RET-1131

Added skeleton pages for the working and notice employment details paths.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/controllers/start_date/StartDateController.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  }
+  public post = (req: AppRequest, res: Response): void => {
+    setUserCase(req, this.form);
+    let redirectUrl = '';
+    if (req.session.userCase.isStillWorking === StillWorking.WORKING) {
+      redirectUrl = PageUrls.NOTICE_PERIOD;
+    } else if (req.session.userCase.isStillWorking === StillWorking.NOTICE) {
+      redirectUrl = PageUrls.NOTICE_END;
+    } else if (req.session.userCase.isStillWorking === StillWorking.NO_LONGER_WORKING) {
+      redirectUrl = PageUrls.HOME;
```
Comment (marcjudge) [inline src/main/controllers/start_date/StartDateController.ts:24]: Please put a TODO in here so it's clear this needs to be wired up

Comment (lvo700) [reply src/main/controllers/start_date/StartDateController.ts:25]: Added TODO

File: src/main/controllers/start_date/StartDateController.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    } else {
+      redirectUrl = PageUrls.START_DATE;
```
[the remark this answers is not in the document]
Comment (lvo700) [reply src/main/controllers/start_date/StartDateController.ts]: Removed redirect to START_DATE as it isn't possible to get to this page without having selected one of the radios in the previous page.

File: src/main/definitions/case.ts [hunk 3]
```diff
@@ -35,13 +35,20 @@ export interface Case {
   noticePeriod?: YesOrNo;
   noticePeriodLength?: string;
   noticePeriodUnit?: WeeksOrMonths;
+  isStillWorking: string;
```
[the remark this answers is not in the document]
Comment (lvo700) [reply src/main/definitions/case.ts]: Changed to the enum

**Review by brendanmchugh — APPROVED** [review]

**Pull request merged**